subprojects/emcee/client/src/main/kotlin/com/avito/emcee/client/internal/JobWaiter.kt

```diff
@@ -16,11 +16,15 @@ internal class JobWaiter(
     @ExperimentalTime
     suspend fun wait(job: Job, timeout: Duration) {
         withTimeout(timeout) {
-            do {
-                // TODO handle error
-                val status = queueApi.jobStatus(JobStatusBody(id = job.id))
-                delay(10.seconds) // TODO how often? to config
-            } while (status.queueState.dequeuedBucketCount + status.queueState.enqueuedBucketCount != 0)
+            while (getTestsCount(job) > 0) {
+                delay(5.seconds) // TODO how often? to config
```

Comment:
Can we subscribe to changes in the queue?

Reply:
Now, we can't do it for now. But it could be possible in the future.